config/teleop.yaml

```diff
@@ -0,0 +1,50 @@
+teleop:
+  constants:
+    track_radius: 10.0 # Meters
```

Comment:
Should've said earlier, but can the comment also discuss what these actually mean? The variable name is not clear